src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ManagedInstanceKeyVaultKeyTests.ps1

```diff
@@ -0,0 +1,135 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+		
+$mangedInstanceRg = "MlAndzic_RG"
```

Comment:
How long does it take to create a managed instance? We like to avoid pre test setup as much as possible.

Reply:
From my last conversation with the managed instance team it takes hours to create one.

Reply:
Okay, can you give detailed instructions on how to properly set up the managed instance you need to run this test successfully?

Reply:
I can ask Manged Instance team to help provide this information, they just advised me to use their test instance